packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -118,6 +119,20 @@ class BuildInfo {
   /// will be written for code size profiling.
   final String codeSizeDirectory;
 
+  /// Whether to enable the Gradle daemon when performing an Android build.
+  ///
+  /// Starting the daemon is the default behavior of the gradle wrapper script created
```

Comment:
Should this go in the help text for the flag?

Reply:
Done